src/main/java/dev/joopie/jambot/repository/album/AlbumRepository.java

```diff
@@ -0,0 +1,10 @@
+package dev.joopie.jambot.repository.album;
+
+import dev.joopie.jambot.model.album.Album;
+import org.springframework.data.repository.ListCrudRepository;
+
+import java.util.Optional;
+
+public interface AlbumRepository extends ListCrudRepository<Album, Long> {
```

Comment:
Werkt dit trouwens zonder de `@Repository` annotatie? Normaal moet je die nog annoteren, bijzonder.

Reply:
Hmm. Good spot. I will add this quickly